src/Files.App/MainWindow.xaml.cs

```diff
@@ -247,11 +248,25 @@ async Task PerformNavigationAsync(string payload, string selectItem = null)
 				}
 
 				var generalSettingsService = Ioc.Default.GetService<IGeneralSettingsService>();
+
+				double boundsWidth = 0;
+				try
+				{
+					boundsWidth = Bounds.Width;
+				}
+				catch (Exception ex)
+				{
+					// Handle exception in case WinUI Windows is closed
+					// (see https://github.com/files-community/Files/issues/15599)
+
+					App.Logger.LogWarning(ex, ex.Message);
```

Comment:
```suggestion
					App.Logger.LogWarning(ex, ex.Message);
					return;
```
In this case, I think the navigation should be aborted.